Website/Licenses/Cake (MIT).txt.resources

```diff
@@ -0,0 +1,30 @@
+﻿PROJECT: Cake
+
+WHERE WE USE IT: Used at multiple places in our build process, for instance at https://github.com/dnnsoftware/Dnn.Platform/blob/development/build.cake
+
+LICENTE TYPE: MIT
```

Comment:
typo (looks like it's present in all files): LICENTE/LICENSE

Reply:
Correct, it was copy pasted from a template, I will go correct that

Reply:
Thanks for spotting that, it is now corrected.